src/DbSchema/SchemaHasherMysql.php

```diff
@@ -76,10 +76,8 @@ public function hashDb(): string
 
             try {
                 $result = $this->connection->query($query);
-                if ($result instanceof \mysqli_result) {
```

Comment:
I think in case of a invalid query or similar, the result will not be a object and we might run into a fatal error after this change?

Reply:
Without this change, we get a PHPStan warning:

> Instanceof between mysqli_result<array<string, mixed>> and mysqli_result will always evaluate to true.  

My understanding was that this change allows us to parse the query string on line 42, and detects that we can't get a boolean return value from this query.

If you'd prefer to keep the check, I'm happy to add it to the ignored warnings in phpstan.neon.dist instead.

Reply:
I think we should leave the check here. I don't see how the query in line 42 is related to the chang regarding queries without a FROM.

in case it really changes the inference for said query, this would mean we miss a new unit test for this scenario

Reply:
This is fixed (reverted to the original form with the type check) now that we are explicitly handling the case where there is no FROM clause. It was erroneously handling other unsupported FROM clause expressions as if they were the no FROM clause case. Thanks for the careful review!